compiler/src/dmd/doc.d

```diff
@@ -35,6 +35,7 @@ import dmd.dsymbol;
 import dmd.dsymbolsem;
 import dmd.dtemplate;
 import dmd.errors;
+import dmd.errorsink;
```

Comment:
Why do we add this import if we're just passing a variable without ever referencing its type ?

Reply:
removed